storage-node/src/services/queryNode/queries/queries.graphql

```diff
@@ -3,19 +3,62 @@ fragment StorageBucketIds on StorageBucket {
 }
 
 query getStorageBuckets {
-  storageBuckets(where: { operatorStatus: { isTypeOf_eq: "StorageBucketOperatorStatusActive" } }) {
+  storageBuckets(
+    where: {
+      operatorStatus: { isTypeOf_eq: "StorageBucketOperatorStatusActive" }
+      operatorMetadata: {
+        OR: [
+          { nodeOperationalStatus_isNull: true }
+          { nodeOperationalStatus: { isTypeOf_eq: "NodeOperationalStatusNormal" } }
+        ]
+      }
+    }
+  ) {
     ...StorageBucketIds
   }
 }
 
-query getStorageBucketsByWorkerId($workerId: Int!) {
+query getStorageBucketsByWorkerId($workerId: BigInt!) {
   storageBuckets(
-    where: { operatorStatus: { isTypeOf_eq: "StorageBucketOperatorStatusActive", workerId_eq: $workerId } }
+    where: {
+      operatorStatus: { isTypeOf_eq: "StorageBucketOperatorStatusActive", workerId_eq: $workerId }
+      operatorMetadata: {
+        OR: [
+          { nodeOperationalStatus_isNull: true }
+          { nodeOperationalStatus: { isTypeOf_eq: "NodeOperationalStatusNormal" } }
+        ]
+      }
```

Comment:
Same comment here as with the query in distribution node

Reply:
Addressed